daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/TransactionCoder.scala

```diff
@@ -141,112 +141,119 @@ object TransactionCoder {
     * @tparam Cid contract id type
     * @return protocol buffer format node
     */
-  private[lf] def encodeNode[Nid, Cid](
+  private[lf] def encodeNodes[Nid, Cid](
```

Comment:
Isn't this function only encoding a single node? If so, the plural "s" doesn't make too much sense.

Reply:
good catch.